modules/ROOT/pages/_partials/app-security/using-the-admin-ui.adoc

```diff
@@ -0,0 +1,58 @@
+= Accessing the console
+
+The {app-security-service} Console is viewed by opening the console URL in a browser. This URL is seen in the list of available services in the Mobile Developer Console (MDC) after the {app-security-service} Service has been provisioned.
+
+= Console overview
+
+The {app-security-service} Console allows you to monitor your applications and the versions of those applications.
```

Comment:
```suggestion
The {app-security-service} Console allows you to monitor applications and their respective versions.
```

Saying "allows you to monitor **your** applications" might lead a user to believe that this is for monitoring their own installed apps.